src/services/EmailService.ts

```diff
@@ -52,13 +55,40 @@ export class EmailService implements IEmailService {
     };
 
     private sendRawEmail = async (params: RawEmailParams) => {
+        const unsignedMessage = await openpgp.createCleartextMessage({
+            text: params.text
+        });
+
+        const privateKeyArmored = fs.readFileSync(
+            path.join(__dirname, "../../") + "/test_idem_com_au.asc",
+            { encoding: "utf8" }
+        );
+
+        const privateKeys = await openpgp.readPrivateKeys({
+            armoredKeys: privateKeyArmored
+        });
+
+        const passphrase = process.env.PGP_PASSPHRASE;
+
+        const privateKey = await openpgp.decryptKey({
+            privateKey: privateKeys[0],
+            passphrase
+        });
+
+        const cleartextMessage = await openpgp.sign({
+            message: unsignedMessage,
+            signingKeys: privateKey
+        });
+
+        console.log(cleartextMessage);
```

Comment:
is this meant to be left over? is there not a logging suite?

Reply:
@xiangzhe0812 lets remove.  Good catch @brettvniekerk